### PR TITLE
Fixes - #521 : Created the Sentiment Analysis App

### PR DESCRIPTION
**Pull request by daminisatya**

### CODE REVIEW

**Comment by daminisatya** [comment]
selection_091
selection_092

**Comment by daminisatya** [comment]
@Orbiter Please review.

File: html/apps/sentimentVisualizer/index.html [hunk 1]
```diff
@@ -51,22 +51,26 @@
...
             </div>
         </div>
+        <!-- <script>
+            var app = angular.module('visualizerPage', []);
+            app.controller('Controller', ['$scope', function($scope) {
+            $scope.redirect = function(){
+                window.location = "/visualize.html";
+            }
+        }]);
+        </script> -->
```
Comment (sudheesh001) [inline html/apps/sentimentVisualizer/index.html]: I think the comments can be dropped. 😄

**Comment by daminisatya** [comment]
@sudheesh001 removed the comments.

**Pull request merged**